packages/asc-core/src/components/RowStyle/RowStyle.ts

```diff
@@ -0,0 +1,144 @@
+import { Color } from 'csstype'
+import styled, { css } from '../../styled-components'
+import { fromTheme, valueFromObject } from '../../utils'
+import { Theme } from '../../theme'
+import layouts from '../../theme/default/layouts'
+import {
+  mediaQuery,
+  min,
+  max,
+  margin,
+  columns,
+  spacerWidth,
+  gutter,
+} from '../../utils/grid'
+
+type FlexJustify =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly'
+type FlexAlign = 'flex-start' | 'center' | 'flex-end' | 'stretch' | 'baseline'
```

Comment:
perhaps you could put these Flex types to a generic file? Since you've declared the same type in ColumnStyle.ts